ocaml/xapi/storage_migrate.ml

```diff
@@ -1383,14 +1376,14 @@ let with_task_and_thread ~dbg f =
 
 let copy ~dbg ~sr ~vdi ~url ~dest ~verify_dest =
   with_task_and_thread ~dbg (fun task ->
-      MigrateLocal.copy_into_sr ~task ~dbg:dbg.Debug_info.log ~sr ~vdi ~url
-        ~dest ~verify_dest
+      MigrateLocal.copy_into_sr ~task ~dbg:(Debug_info.to_string dbg) ~sr ~vdi
```

Comment:
This is a different string now. The `dbg.Debug_info.log` string was used here (and below), because the functions further down do not yet handle dbg strings that have both the "log" and "tracing" data combined, which can then end up in the "task" part of log lines. Or has that changed now?

Reply:
that is quite true. But line 733, which I deleted, has `let dbg = dbg_and_tracing_of_trask task`  in `start` function will mean that anything below that line in the `start` function will be using a `dbg` parameter with "log" and "dbg" in them. Is this intended?

Reply:
I had a look at the following functions which use `dbg`, it seems that they are all SMAPIv2 calls, which handles tracing already. If they are what you were referring to as functions further down?